Revert switch from ansi_term (which is in Debian) to nu-ansi-term (which is
not in Debian). 

This patch is based on a revert of upstream commit
f09bc43c3f13e6df856b809d47544e7e846a6192

Index: tracing-subscriber/src/filter/env/builder.rs
===================================================================
--- tracing-subscriber.orig/src/filter/env/builder.rs
+++ tracing-subscriber/src/filter/env/builder.rs
@@ -209,15 +209,15 @@ impl Builder {
         }
 
         if !disabled.is_empty() {
-            #[cfg(feature = "nu_ansi_term")]
-            use nu_ansi_term::{Color, Style};
+            #[cfg(feature = "ansi_term")]
+            use ansi_term::{Color, Style};
             // NOTE: We can't use a configured `MakeWriter` because the EnvFilter
             // has no knowledge of any underlying subscriber or collector, which
             // may or may not use a `MakeWriter`.
             let warn = |msg: &str| {
-                #[cfg(not(feature = "nu_ansi_term"))]
+                #[cfg(not(feature = "ansi_term"))]
                 let msg = format!("warning: {}", msg);
-                #[cfg(feature = "nu_ansi_term")]
+                #[cfg(feature = "ansi_term")]
                 let msg = {
                     let bold = Style::new().bold();
                     let mut warning = Color::Yellow.paint("warning");
@@ -227,9 +227,9 @@ impl Builder {
                 eprintln!("{}", msg);
             };
             let ctx_prefixed = |prefix: &str, msg: &str| {
-                #[cfg(not(feature = "nu_ansi_term"))]
+                #[cfg(not(feature = "ansi_term"))]
                 let msg = format!("{} {}", prefix, msg);
-                #[cfg(feature = "nu_ansi_term")]
+                #[cfg(feature = "ansi_term")]
                 let msg = {
                     let mut equal = Color::Fixed(21).paint("="); // dark blue
                     equal.style_ref_mut().is_bold = true;
@@ -240,9 +240,9 @@ impl Builder {
             let ctx_help = |msg| ctx_prefixed("help:", msg);
             let ctx_note = |msg| ctx_prefixed("note:", msg);
             let ctx = |msg: &str| {
-                #[cfg(not(feature = "nu_ansi_term"))]
+                #[cfg(not(feature = "ansi_term"))]
                 let msg = format!("note: {}", msg);
-                #[cfg(feature = "nu_ansi_term")]
+                #[cfg(feature = "ansi_term")]
                 let msg = {
                     let mut pipe = Color::Fixed(21).paint("|");
                     pipe.style_ref_mut().is_bold = true;
Index: tracing-subscriber/src/fmt/format/mod.rs
===================================================================
--- tracing-subscriber.orig/src/fmt/format/mod.rs
+++ tracing-subscriber/src/fmt/format/mod.rs
@@ -46,7 +46,7 @@ use tracing_core::{
 use tracing_log::NormalizeEvent;
 
 #[cfg(feature = "ansi")]
-use nu_ansi_term::{Color, Style};
+use ansi_term::{Colour, Style};
 
 #[cfg(feature = "json")]
 mod json;
@@ -101,7 +101,7 @@ pub use pretty::*;
 ///   does not support ANSI escape codes (such as a log file), and they should
 ///   not be emitted.
 ///
-///   Crates like [`nu_ansi_term`] and [`owo-colors`] can be used to add ANSI
+///   Crates like [`ansi_term`] and [`owo-colors`] can be used to add ANSI
 ///   escape codes to formatted output.
 ///
 /// * The actual [`Event`] to be formatted.
@@ -189,7 +189,7 @@ pub use pretty::*;
 /// [implements `FormatFields`]: super::FmtContext#impl-FormatFields<'writer>
 /// [ANSI terminal escape codes]: https://en.wikipedia.org/wiki/ANSI_escape_code
 /// [`Writer::has_ansi_escapes`]: Writer::has_ansi_escapes
-/// [`nu_ansi_term`]: https://crates.io/crates/nu_ansi_term
+/// [`ansi_term`]: https://crates.io/crates/ansi_term
 /// [`owo-colors`]: https://crates.io/crates/owo-colors
 /// [default formatter]: Full
 pub trait FormatEvent<S, N>
@@ -1484,11 +1484,11 @@ impl<'a> fmt::Display for FmtLevel<'a> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         if self.ansi {
             match *self.level {
-                Level::TRACE => write!(f, "{}", Color::Purple.paint(TRACE_STR)),
-                Level::DEBUG => write!(f, "{}", Color::Blue.paint(DEBUG_STR)),
-                Level::INFO => write!(f, "{}", Color::Green.paint(INFO_STR)),
-                Level::WARN => write!(f, "{}", Color::Yellow.paint(WARN_STR)),
-                Level::ERROR => write!(f, "{}", Color::Red.paint(ERROR_STR)),
+                Level::TRACE => write!(f, "{}", Colour::Purple.paint(TRACE_STR)),
+                Level::DEBUG => write!(f, "{}", Colour::Blue.paint(DEBUG_STR)),
+                Level::INFO => write!(f, "{}", Colour::Green.paint(INFO_STR)),
+                Level::WARN => write!(f, "{}", Colour::Yellow.paint(WARN_STR)),
+                Level::ERROR => write!(f, "{}", Colour::Red.paint(ERROR_STR)),
             }
         } else {
             match *self.level {
Index: tracing-subscriber/src/fmt/format/pretty.rs
===================================================================
--- tracing-subscriber.orig/src/fmt/format/pretty.rs
+++ tracing-subscriber/src/fmt/format/pretty.rs
@@ -14,7 +14,7 @@ use tracing_core::{
 #[cfg(feature = "tracing-log")]
 use tracing_log::NormalizeEvent;
 
-use nu_ansi_term::{Color, Style};
+use ansi_term::{Colour, Style};
 
 /// An excessively pretty, human-readable event formatter.
 ///
@@ -143,11 +143,11 @@ impl Default for Pretty {
 impl Pretty {
     fn style_for(level: &Level) -> Style {
         match *level {
-            Level::TRACE => Style::new().fg(Color::Purple),
-            Level::DEBUG => Style::new().fg(Color::Blue),
-            Level::INFO => Style::new().fg(Color::Green),
-            Level::WARN => Style::new().fg(Color::Yellow),
-            Level::ERROR => Style::new().fg(Color::Red),
+            Level::TRACE => Style::new().fg(Colour::Purple),
+            Level::DEBUG => Style::new().fg(Colour::Blue),
+            Level::INFO => Style::new().fg(Colour::Green),
+            Level::WARN => Style::new().fg(Colour::Yellow),
+            Level::ERROR => Style::new().fg(Colour::Red),
         }
     }
 
Index: tracing-subscriber/src/lib.rs
===================================================================
--- tracing-subscriber.orig/src/lib.rs
+++ tracing-subscriber/src/lib.rs
@@ -10,7 +10,7 @@
 //! `tracing-subscriber` is intended for use by both `Subscriber` authors and
 //! application authors using `tracing` to instrument their applications.
 //!
-//! *Compiler support: [requires `rustc` 1.50+][msrv]*
+//! *Compiler support: [requires `rustc` 1.49+][msrv]*
 //!
 //! [msrv]: #supported-rust-versions
 //!
@@ -138,7 +138,7 @@
 //! ## Supported Rust Versions
 //!
 //! Tracing is built against the latest stable release. The minimum supported
-//! version is 1.50. The current Tracing version is not guaranteed to build on
+//! version is 1.49. The current Tracing version is not guaranteed to build on
 //! Rust versions earlier than the minimum supported version.
 //!
 //! Tracing follows the same compiler support policies as the rest of the Tokio
Index: tracing-subscriber/Cargo.toml
===================================================================
--- tracing-subscriber.orig/Cargo.toml
+++ tracing-subscriber/Cargo.toml
@@ -65,8 +65,8 @@ harness = false
 version = "0.1.0"
 optional = true
 
-[dependencies.nu-ansi-term]
-version = "0.46.0"
+[dependencies.ansi_term]
+version = "0.12.1"
 optional = true
 
 [dependencies.once_cell]
@@ -174,7 +174,7 @@ version = "0.1.3"
 alloc = []
 ansi = [
     "fmt",
-    "nu-ansi-term",
+    "ansi_term",
 ]
 default = [
     "smallvec",
